README.md

```diff
@@ -139,6 +139,39 @@ only really makes sense if you're calling Grover directly (and not via middlewar
 Grover.new('<some URI with basic authentication', username: 'the username', password: 'super secret').to_pdf
 ```
 
+#### Adding cookies
+If you need to set cookies to your new page instance, just pass them along like this:
```

Comment:
I'd suggest adding a note here to make it clear that the `name` and `value` attributes are required, and that other properties are optional. Maybe also link to https://github.com/puppeteer/puppeteer/blob/master/docs/api.md#pagesetcookiecookies ?